recipes/7bitconf/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import get, copy, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class SevenBitConfConan(ConanFile):
+    name = "7bitconf"
+    homepage = "https://github.com/7bitCoder/7bitConf"
+    description = "7bitConf is a simple C++17 centralized configuration provider library."
+    topics = ("cpp17", "configuration", "provider", "configuration-files")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "header_only": [True, False],
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "header_only": False,
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+    
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "14",
+            "msvc": "192",
+            "gcc": "8",
+            "clang": "7",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
+    def configure(self):
+        if self.options.get_safe("shared") or self.options.header_only:
+            self.options.rm_safe("fPIC")
+        if self.options.header_only:
+            self.options.rm_safe("shared")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("taocpp-json/1.0.0-beta.13", transitive_headers=True, transitive_libs=True)
```

Comment:
Are you completely sure that the transitivity is required?
- The headers transitivity is more common, if your public package headers include headers from taocpp-json, then this is necessary. I guess this is the case
- The libs transitivity should be way more exceptional, but wouldn't be necessary to be specified for most regular libraries. So if it is really required, then it would be necessary to add some comment describing the reason.

Reply:
Thank you, taocpp json library is header only and is included in public heareds so in this case I guess that transitive_libs is not necesary, I have removed it, now only transitive_headers option is set to True.